text/src/autogluon/text/automm/models/utils.py

```diff
@@ -0,0 +1,190 @@
+from typing import Optional, Union, Tuple, List, Dict
+from torch import nn
+
+
+def init_weights(module: nn.Module):
+    if isinstance(module, nn.Embedding):
+        nn.init.xavier_normal_(module.weight)
+    elif isinstance(module, nn.Linear):
+        nn.init.xavier_uniform_(module.weight)
+        if module.bias is not None:
+            nn.init.zeros_(module.bias)
+    elif isinstance(module, nn.LayerNorm):
+        module.bias.data.zero_()
+        module.weight.data.fill_(1.0)
```

Comment:
This should account what activations are used - if ReLU is used then `kaiming_normal_` should be used; if Leaky ReLU, then `kaiming_uniform_` is better at preventing exploding/collapsing gradients. Xavier initialization is better for sigmoid activation function. I know transformers heavily prefer sigmoid gates, but if this code is used in other architectures with ReLU, it's not an optimal choice - maybe add documentation when it supposed to be used.

Reply:
Good suggestion! Will add some introduction in the docstrings.